<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="MessageWindow">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionCreate_HDF5_file"/>
    <addaction name="actionAdd_to_HDF5_file"/>
    <addaction name="actionAdd_levels_to_HDF5_file"/>
    <addaction name="actionPlot"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuOpen_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_HDF5_file">
   <property name="text">
    <string>Create HDF5 file</string>
   </property>
  </action>
  <action name="actionAdd_to_HDF5_file">
   <property name="text">
    <string>Add to HDF5 file</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
  <action name="actionAdd_levels_to_HDF5_file">
   <property name="text">
    <string>Add levels</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
